supervisor/homeassistant/websocket.py

```diff
@@ -74,7 +75,7 @@ async def async_send_message(self, message: dict[str, Any]) -> None:
         self._message_id += 1
         _LOGGER.debug("Sending: %s", message)
         try:
-            await self._client.send_json(message)
+            await self._client.send_json(message, dumps=json_dumps)
```

Comment:
This is the only one that matters but good to be consistent and use the same everywhere